### PR TITLE
Add optional dropdownStyle to ActionMenu for consistency

### PR DESCRIPTION
**Pull request by jangmi-jo**
This commit adds optional dropdownStyle to ActionMenu like SingleSelect has.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 206176aa714a025515f9d4945553ad763d1057c5

https://deploy-preview-371--wonder-blocks.netlify.com

File: packages/wonder-blocks-dropdown/generated-snapshot.test.js [hunk 1]
```diff
@@ -227,6 +227,66 @@ describe("wonder-blocks-dropdown", () => {
         const {View} = require("@khanacademy/wonder-blocks-core");
         const {StyleSheet} = require("aphrodite");
 
+        const styles = StyleSheet.create({
+            row: {
+                flexDirection: "row",
+            },
+            dropdown: {
+                maxHeight: 200,
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/generated-snapshot.test.js]: It's too bad this doesn't show up in the snapshot.

Comment (somewhatabstract) [reply packages/wonder-blocks-dropdown/generated-snapshot.test.js]: What do you mean, "it doesn't show up"? We have some cunning code to get aphrodite styles into the snapshot so I'm concerned there's a bug.

Comment (kevinbarabash) [reply packages/wonder-blocks-dropdown/generated-snapshot.test.js]: It doesn't show up b/c the component doesn't show the dropdown until the user clicks on it.

**Pull request merged**